### PR TITLE
Add No-Login Web Apps

### PR DESCRIPTION
**Pull request by aviaryan**
**https://github.com/aviaryan/awesome-no-login-web-apps**

**It is a curated list of awesome web apps that work without requesting you to create an account. This list will help users to do common tasks without having to download a software or create web accounts.**

PS - I have not added a banner image to the list yet, the reason being I am not good in design. I would love a PR about it. I am still going on with this PR because I saw some lists in this list of lists without a logo/illustration. 

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> It is a curated list of awesome web apps that work without requesting you to create an account. This list will help users to do common tasks without having to download a software or create web accounts.

What problem does it solve though? I honestly rather create an account in a great web app then use a crappy one without login.

**Comment by aviaryan** [comment]
The main motivation for this is like suppose you want to do a certain task, let's say plot a graph for a function to see its behaviour. You come to no-login-web-apps repo, look up the index and quickly find something that works for you, out of the box. No logins, no registrations. Of course, if you will be doing this too often, you might want to create an account on that site.

> I honestly rather create an account in a great web app then use a crappy one without login

**From README:** Most of the apps mentioned here do provide some kind of a user account feature to save history and progress. But as already stated before, a user account is not necessary to use the main features of these apps.

Maybe I should add a marker for sites that allow logins ?

----

Considering the above explanation, I think my PR is suitable for this project. @sindresorhus What do you think?

**Comment by jedahan** [comment]
I also see this list as a great resource on finding applications that have the potential to be easily distributed, offline first, generally (though not always) small codebase...

File: readme.md [hunk 1]
```diff
@@ -482,6 +482,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Web Design](https://github.com/nicolesaidy/awesome-web-design) - For digital designers.
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter) - Load testing and performance measurement tool.
 - [Creative Coding](https://github.com/terkelg/awesome-creative-coding) - Programming something expressive instead of something functional.
+- [No Login Web Apps](https://github.com/aviaryan/awesome-no-login-web-apps) - Web apps that work without login
```
Comment (sindresorhus) [inline readme.md]: Should it be `No-Login Web Apps`? I'm not sure, but sounds better. You're also missing a `.` at the end.

**Comment by sindresorhus** [comment]
> @sindresorhus What do you think?

:+1:

> only has awesome items with descriptions

Noticed a few items are missing description.

> Each link description starts with an uppercase character and ends with a period.

Not followed.

> Drop all the A / An prefixes in the descriptions.

Not followed.

> Consistent naming. For example, Node.js, not NodeJS or node.js.

For example, I see both `Youtube` and `youtube` used, but neither are correct. It's `YouTube`.

File: readme.md (hunk 1 above)
Comment (aviaryan) [inline readme.md]: :+1:

**Comment by aviaryan** [comment]
@sindresorhus I have updated the list with the improvements you suggested. Can you please have a look. Thanks 😊

**Review by AngSolids — APPROVED** [review]

**Comment by AngSolids** [comment]
Can you forget to stroger you're talking bullet

**Pull request merged**